packages/apps/graph/prisma/schema.prisma

```diff
@@ -0,0 +1,143 @@
+
+  @@id([block, idx, requestkey])
+  @@index([height(sort: Desc), chainid, idx], map: "events_height_chainid_idx")
+  @@index([requestkey])
+  @@map("events")
+}
+
+model Minerkey {
+  block  String @db.VarChar
+  key    String @db.VarChar
+  blocks Block  @relation(fields: [block], references: [hash], onDelete: NoAction, onUpdate: NoAction)
+
+  @@id([block, key])
+  @@map("minerkeys")
+}
+
+model Signer {
+  addr       String? @db.VarChar
+  caps       Json
+  idx        Int
+  pubkey     String  @db.VarChar
+  requestkey String  @db.VarChar
+  scheme     String? @db.VarChar
+  sig        String  @db.VarChar
+
+  @@id([idx, requestkey])
+  @@map("signers")
+}
+
+model Transaction {
+  badresult    Json?
+  block        String   @db.VarChar
+  chainid      BigInt
+  code         String?  @db.VarChar
+  continuation Json?
+  creationtime DateTime @db.Timestamptz(6)
+  data         Json?
+  gas          BigInt
+  gaslimit     BigInt
+  gasprice     Float
+  goodresult   Json?
+  height       BigInt
+  logs         String?  @db.VarChar
+  metadata     Json?
+  nonce        String   @db.VarChar
+  num_events   BigInt?
+  pactid       String?  @db.VarChar
+  proof        String?  @db.VarChar
+  requestkey   String   @db.VarChar
+  rollback     Boolean?
+  sender       String   @db.VarChar
+  step         BigInt?
+  ttl          BigInt
+  txid         BigInt?
+  blocks       Block    @relation(fields: [block], references: [hash], onDelete: NoAction, onUpdate: NoAction)
+
+  @@id([block, requestkey])
+  @@index([height])
+  @@index([requestkey])
+  @@map("transactions")
+}
+
+model Transfer {
+  amount     Decimal @db.Decimal
+  block      String  @db.VarChar
+  chainid    BigInt
+  from_acct  String  @db.VarChar
+  height     BigInt
+  idx        BigInt
+  modulehash String  @db.VarChar
+  modulename String  @db.VarChar
+  requestkey String  @db.VarChar
+  to_acct    String  @db.VarChar
+  blocks     Block   @relation(fields: [block], references: [hash], onDelete: NoAction, onUpdate: NoAction)
+
+  @@id([block, chainid, idx, modulehash, requestkey])
+  @@index([from_acct, height(sort: Desc), idx], map: "transfers_from_acct_height_idx")
+  @@index([to_acct, height(sort: Desc), idx])
+  @@map("transfers")
+}
+
+/// The underlying table does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying table does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying table does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying table does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying table does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
```

Comment:
Was this left intentionally?

Reply:
It's being generated by the prisma cli. Will remove at some point.